crypto/cipher/aes_icm.c

```diff
@@ -311,16 +292,15 @@ static void srtp_aes_icm_advance_ismacryp (srtp_aes_icm_ctx_t *c, uint8_t forIsm
  *  - fill buffer then add in remaining (< 16) bytes of keystream
  */
 
-static srtp_err_status_t srtp_aes_icm_encrypt_ismacryp (srtp_aes_icm_ctx_t *c,
-                                                 unsigned char *buf, unsigned int *enc_len,
-                                                 int forIsmacryp)
+static srtp_err_status_t srtp_aes_icm_encrypt (srtp_aes_icm_ctx_t *c,
```

Comment:
This function also throws an error about incompatible pointer. The signature to match is:

~~~.c
typedef srtp_err_status_t (*srtp_cipher_encrypt_func_t)
    (void *state, uint8_t *buffer, unsigned int *octets_to_encrypt);
~~~

Just changing '*c' to void did _not_ help :-) This worked and is used elsewhere:

~~~.c
static srtp_err_status_t srtp_aes_icm_encrypt (void *cv,
                                               uint8_t *buf, unsigned int *enc_len)
{
    (srtp_aes_icm_ctx_t*)c = (srtp_aes_icm_ctx_t*) cv;
    // ...
~~~